Cargo.lock

```diff
@@ -2843,8 +2843,8 @@ dependencies = [
  "itp-top-pool",
  "itp-types",
  "itp-utils",
- "jsonrpc-core 16.1.0",
- "jsonrpc-core 18.0.0",
+ "jsonrpc-core 18.0.0 (registry+https://github.com/rust-lang/crates.io-index)",
+ "jsonrpc-core 18.0.0 (git+https://github.com/scs/jsonrpc?branch=no_std_v18)",
```

Comment:
Is there a good reason to have both? Shouldn't we depend uniquely on the one that allows no_std?

Reply:
The name is actually misleading. It's not completely no_std compatible. Because of the futures dependency, we were not able to make it completely no_std compatible: https://github.com/scs/jsonrpc/blob/no_std_v18/core/Cargo.toml#L23

I only updated it to keep up to date with parity, but I didn't clean up anything. Which would be good, because we could, for example, get rid of sp_std. But I think it's tolerable.